app/src/types/generic/routes.ts

```diff
@@ -11,6 +11,8 @@ export enum routes {
     DOWNLOAD_DOCUMENTS = '/search/results',
     DELETE_DOCUMENTS = '/search/results/delete',
 
+    LLOYD_GEORGE = '/lloyd-george-record',
```

Comment:
Has this URL been agreed with ? Seems a bit odd to not have any more sub-directories in the path as you will have navigated through some pages by this